thirdeye-server/src/main/java/ai/startree/thirdeye/resources/AlertResource.java

```diff
@@ -257,4 +267,26 @@ private long safeEndTime(final @Nullable Long endTime) {
     }
     return endTime;
   }
+
+  @GET
+  @Timed
+  @Path("{id}/stats")
+  @Produces(MediaType.APPLICATION_JSON)
+  public Response getAnalytics(
+      @ApiParam(hidden = true) @Auth ThirdEyePrincipal principal,
+      @PathParam("id") final Long id,
+      @Context UriInfo uriInfo
```

Comment:
Use `@QueryParam`. if @PathParam exists, so should QueryParam. no?

Reply:
Done